diff --git a/api_client/src/apis/default_api.rs b/api_client/src/apis/default_api.rs
index 53a7d45..15f791d 100644
--- a/api_client/src/apis/default_api.rs
+++ b/api_client/src/apis/default_api.rs
@@ -2042,7 +2042,7 @@ pub async fn suggest_query(
     let local_var_uri_str = format!(
         "{}/api/queries/{datasource_type}/suggestions",
         configuration.base_path,
-        datasource_type = datasource_type
+        datasource_type = serde_json::to_string(&datasource_type)?
     );
     let mut local_var_req_builder =
         local_var_client.request(reqwest::Method::GET, local_var_uri_str.as_str());
